policies/lib/all_attestations.rego

```diff
@@ -0,0 +1,10 @@
+package lib
+
+#
+# A list of all attestations
+#
+all_attestations = attestations {
+	attestations := [attestation |
+		attestation := {"data": data.attestations[_]}
+	]
+}
```

Comment:
There's still the possibility that we should just scrap this file entirely and just use data.attestations instead of lib.all_attestations, and replace the `att.data` with just `att` (for example).

There might be some value in keep it in case we someday decide we want fields in the hash other than "data", (like before we had rekor_host and log_index), and maybe someday we might lift up a sha or something, but right now I think this isn't bringing much value.

Is there another reason to convert ["..."] into `[{data: "..."}]`?

Reply:
Nope. It was just as a way to provide additional metadata for each attestation. Let's get rid of it.

Reply:
Done.